src/api.v2/model/Experiment.js

```diff
@@ -43,20 +44,26 @@ class Experiment extends BasicModel {
     ];
 
     const aliasedExperimentFields = fields.map((field) => `e.${field}`);
-    function mainQuery() {
-      this.select([...aliasedExperimentFields, 'm.key'])
+
+    const result = await collapseKeyIntoArray(
```

Comment:
not sure if this is an improvement, it makes it harder to read the actual query

Reply:
The main change is to stop using mainQuery as a function, I will keep that change but I can make `const mainQuery = ...`